content/api/hub/plugin_spin_kinesis_trigger.md

```diff
@@ -0,0 +1,77 @@
+title = "Kinesis Trigger"
+template = "render_hub_content_body"
+date = "2024-10-04T00:22:56Z"
+content-type = "text/plain"
+tags = ["trigger", "rust", "plugins"]
+
+[extra]
+author = "ogghead"
+type = "hub_document"
+category = "Plugin"
+language = "Rust"
+created_at = "2024-08-31T00:22:56Z"
+last_updated = "2024-10-04T00:22:56Z"
+spin_version = ">=v2.6.0"
+summary = "An experimental Kinesis trigger for Spin apps"
+url = "https://github.com/ogghead/spin-trigger-kinesis"
+keywords = "trigger, kinesis"
+
+---
+
+An experimental plugin that enables AWS Kinesis trigger for Spin applications.
+
+## Installing Plugin
+
+You can install the plugin using the following command:
+
+```sh
+spin plugins install --url https://github.com/ogghead/spin-trigger-kinesis/releases/download/canary/trigger-kinesis.json
```

Comment:
Good point! I listed the canary because (currently) it includes features that aren't in a tagged version, and I saw a lot of trigger plugins on Spin hub doing the same (e.g. https://developer.fermyon.com/hub/preview/plugin_sqs_trigger). My hope is to swap this once the plugin is not under active feature development (planning 0.4 to be a longer term stable release), but happy to swap it now if that's preferred. Let me know your thoughts!

Reply:
Do not really have a strong preference but I think the case with other plugins in the hub might be half of them are not published to the spin plugin registry.